docs/root/configuration/http_filters/ext_authz_filter.rst

```diff
@@ -0,0 +1,51 @@
+.. _config_http_filters_ext_authz:
+
+External Authorization
+======================
+* External authorization service overview :ref:`architecture overview <arch_overview_ext_authz>`
+* :ref:`HTTP filter v2 API reference <envoy_api_msg_config.filter.http.ext_authz.v2alpha.ExtAuthz>`
+
+The external authorization HTTP filter calls an external authorization service to check if the incoming HTTP request is authorized or not.
+If the request is deemed unauthorized then the request will be denied with 403 (Forbidden) response.
+
+It is recommended that this filter is configured first in the filter chain so that requests are authorized prior to rest of the  filters running through the request.
```

Comment:
How about making this more visible? It looks important.

Reply:
:+1: